src/Mod/Path/PathScripts/PathDressupLeadInOut.py

```diff
@@ -33,46 +33,129 @@
 import math
 import copy
 
-from PySide import QtCore, QtGui
+from PySide import QtGui
 
 __doc__ = """LeadInOut Dressup MASHIN-CRC USE ROLL-ON ROLL-OFF to profile"""
 
-if FreeCAD.GuiUp:
-    import FreeCADGui
+from PySide.QtCore import QT_TRANSLATE_NOOP
+
+translate = FreeCAD.Qt.translate
 
-# Qt translation handling
-def translate(text, context="Path_DressupLeadInOut", disambig=None):
-    return QtCore.QCoreApplication.translate(context, text, disambig)
+if False:
+    PathLog.setLevel(PathLog.Level.DEBUG, PathLog.thisModule())
+    PathLog.trackModule(PathLog.thisModule())
+else:
+    PathLog.setLevel(PathLog.Level.INFO, PathLog.thisModule())
 
-PathLog.setLevel(PathLog.Level.INFO, PathLog.thisModule())
 
-movecommands = ['G1', 'G01', 'G2', 'G02', 'G3', 'G03']
-rapidcommands = ['G0', 'G00']
-arccommands = ['G2', 'G3', 'G02', 'G03']
+movecommands = ["G1", "G01", "G2", "G02", "G3", "G03"]
+rapidcommands = ["G0", "G00"]
+arccommands = ["G2", "G3", "G02", "G03"]
```

Comment:
Can we import these from PathGeom since the module is already available?

Reply:
Yeah i wanted to keep this commit laser focused on fixing the translation issue. There is a ton of refactoring i want to do to this module after we go over this hurdle.

Reply:
ok